README.md

```diff
@@ -855,7 +855,7 @@ Future.of(1)
 //> 2
 ```
 
-For comparison, the equivalent with Promises is:
+For comparison, the approximation with Promises is:
```

Comment:
Instead of using "the approximation", let's use "an approximation" across the docs, as there are most likely more than one approximation.